lib/core/Core.ts

```diff
@@ -98,4 +103,21 @@ export default class Core {
     const response = requestHandler.handleResolveRequest(didOrDidDocument);
     return response;
   }
+
+  /**
+   * Handles the get version request. It gets the versions from the dependent services
+   * as well.
+   */
+  public async handleGetVersionRequest(): Promise<ResponseModel> {
```

Comment:
Consider not having `Request` in method name.
```suggestion
  public async getServiceVersion(): Promise<ResponseModel> {
```

Reply:
Leaving this file the way it is and updating the Blockchain.ts to be consistent with the rest of functions within each respective file.